config/db.config.js

```diff
@@ -1,13 +1,12 @@
-// @ts-nocheck
 import mongoose from "mongoose";
 import * as dotenv from "dotenv";
 
 dotenv.config();
 const uri = process.env.MONGO_URI;
 const options = {
   dbName: "data",
-  // useNewUrlParser: true,
-  // useUnifiedTopology: true,
+  useNewUrlParser: true,
```

Comment:
Any reasons why you have uncommented these? I want to learn 😅

Reply:
To avoid MongoDB warnings as below version 5.7

Reply:
Some of these options bring warnings when used😅😅. I have never understood which ones to use and which ones not to use.